dart/pubspec.yaml

```diff
@@ -15,7 +15,7 @@ dependencies:
   uuid: ^2.0.0
 
 dev_dependencies:
-  mockito: ^4.1.2
+  mockito: ^4.1.1
```

Comment:
because of null-safety, this is not working right thru the IDE, downgrading version made it to work